src/utils/postgres-client/index.ts

```diff
@@ -49,7 +49,7 @@ export const pgClient = {
       [id]
     );
     client.release();
-    return rows[0];
```

Comment:
Surprising Typescript is ok with that, `rows` appear to have a very light type

Reply:
indeed, my first thought was "this would never happen in Rust" ahaha :D